### PR TITLE
Update for dmd 2.072.1 release

### PR DESCRIPTION
**Pull request by Calrama**
https://github.com/gentoo/dlang/pull/44 with squashed history. Hopefully, this time it's correct.

### CODE REVIEW

**Comment by Calrama** [comment]
Looking for reference at e.g. https://github.com/gentoo/dlang/commit/2b517e0f3af5f2d7a431f0eb0d0c6d6b4773c8c1 I don't see why it fails again.

File: eclass/dlang-compilers.eclass [hunk 1]
```diff
@@ -29,6 +29,7 @@ dlang-compilers_declare_versions() {
 		[7]="2.069 x86 amd64"
 		[8]="2.070 x86 amd64"
 		[9]="2.071 x86 amd64"
+		[10]="2.072 x86 amd64"
```
Comment (mleise) [inline eclass/dlang-compilers.eclass]: That line should read "[10]="2.072 ~x86 ~amd64". It should match the KEYWORDS on the corresponding dmd ebuild.

**Comment by mleise** [comment]
Merry Christmas. Take a look at https://github.com/gentoo/dlang/pull/45/files#diff-805ab8c260caaf3235070634d40cb3d1R32
I hope that solves most of the remaining issues. Anything error mentioning ldc2 and llvm-3.6 is not from your commit, but a result of changes to Gentoo's llvm ebuilds.

**Comment by Calrama** [comment]
Merry Christmas to you, as well. I changed the keywords in the dmd ebuild since the only reason they weren't "x86 amd64" to begin with was because I copied from an old ebuild before fetching upstream master (and then forgot about it).

**Comment by mleise** [comment]
It's ok. If Gentoo guidelines haven't changed all new ebuilds must pass a 30 day evaluation period in UNSTABLE.

**Comment by Calrama** [comment]
Ah, I'm somewhat out of touch with Gentoo policies (since I'm on Funtoo), but I'll try to remember that.

**Pull request merged**